src/tools/pub/cmd/index.md

```diff
@@ -83,24 +104,40 @@ transformers before invoking the specified script.
 * The [`pub run`](/tools/pub/cmd/pub-run) command invokes a Dart script in your
   package, or in one of its dependencies.
 
-* The [`pub global`](/tools/pub/cmd/pub-global) command lets you work with globally
-  available packages that can be run when you are not currently inside
+* The [`pub global`](/tools/pub/cmd/pub-global) command lets you work with
+  globally available packages that can be run when you are not currently inside
   a package.
 
-## Deployment
+## Building and deploying apps and packages
+
+Pub supports building and deploying apps and packages. Some information
+is specific to web apps or command-line apps.
+
```

Comment:
Let's make this info a little more helpful. Maybe:

With pub you can publish packages, build deployable web apps, and deploy command-line apps.